src/sagemaker_training/process.py

```diff
@@ -39,6 +39,24 @@
 _DEFAULT_BUF_SIZE = 1024 * 64
 
 
+def get_debugger_exception_classes():
+    """Set exception classes"""
+    exception_classes = []
+    try:
+        from smdebug import exceptions
+
+        # list of exceptions debugger wants training toolkit to catch and log
+        exception_classes += [x for x in dir(exceptions) if isclass(getattr(exceptions, x))]
+        print(f"debug: {dir(exceptions)}")
+    except ImportError:
+        logger.info("No exception classes found in smdebugger")
+
+    print(f"debug: {type(exception_classes[0])}")
```

Comment:
remove debug print

Reply:
can you use a more detailed message like:
   "smdistributed.dataparallel not found or "
            "using an older version without custom exceptions."
            "SM training toolkit will track user script error only"